org.eclipse.draw2d/src/org/eclipse/draw2d/graph/CompoundBreakCycles.java

```diff
@@ -77,58 +73,62 @@ private void cycleRemove(NodeList children) {
 			// outDegree - inDegree
 			Node max = findNodeWithMaxDegree(children);
 			if (max != null) {
-				for (int i = 0; i < children.size(); i++) {
-					Node child = (Node) children.get(i);
-					if (child.flag)
+				for (Node child2 : children) {
+					Node child = child2;
```

Comment:
Is there any reason to create this local variable instead of using the loop parameter? I noticed something similar in buildNestingTreeIndices().

Reply:
This is sometimes created by the auto clean-up of Eclipse IDE. I tried to fix it manually. But it looks like I missed a few. Thx for noting.

Reply:
fxed